libs/damap/src/lib/widgets/export-warning-dialog/export-warning-dialog.html

```diff
@@ -3,7 +3,7 @@ <h1 mat-dialog-title>{{'dialog.export.title' | translate}}</h1>
   {{'dialog.export.content' | translate}}
   <div class="container">
     <ng-container>
-      <mat-form-field *ngIf="!funderSupported">
+      <mat-form-field appearance="fill">
```

Comment:
Non-textarea form fields should have appearance="outline" to keep in sync with the rest of the project.